chainio/clients/fireblocks/contract_call.go

```diff
@@ -96,12 +96,19 @@ func NewContractCallRequest(
 			Calldata: calldata,
 		},
 		ReplaceTxByHash: replaceTxByHash,
-		GasPrice:        gasPrice,
 		GasLimit:        gasLimit,
-		MaxFee:          maxFee,
-		PriorityFee:     priorityFee,
-		FeeLevel:        feeLevel,
 	}
+
```

Comment:
Is it caller (of NewContractCallRequest) responsibility to ensure exactly one of the 3 options is set?
In that case it will make sense to validate, or at least log if this assumption doesn't hold

Reply:
The caller doesn't necessarily have to specify any of the options above, in which case Fireblocks will default `feeLevel` to `MEDIUM`